client-app/pages/cart.vue

```diff
@@ -227,23 +233,27 @@ function handleSelectItems(value: { itemIds: string[]; selected: boolean }) {
   }
 }
 
-void (async () => {
-  await forceFetch();
-
-  /**
-   * Send a Google Analytics shopping cart view event.
-   */
-  if (cart.value) {
-    ga.viewCart(cart.value);
-  }
-
-  if (!config.checkout_multistep_enabled) {
-    await initialize();
-  }
-
-  const isXRecommendModuleEnabled = isEnabledXRecommend(XRECOMMEND_ENABLED_KEY);
-  if (isAuthenticated.value && isXRecommendModuleEnabled) {
-    recentlyBrowsedProducts.value = (await recentlyBrowsed())?.products || [];
-  }
-})();
+watch(
+  () => props.cartId,
+  async () => {
+    await forceFetch({ cartId: props.cartId });
```

Comment:
remove watch

Reply:
It will not work without it and I don't see alternatives. Why this is an issue?